### PR TITLE
Set sanitizers options in the current shell too

### PR DESCRIPTION
**Pull request by azat**
/etc/environment will not be read in current shell, so export variables
from that file manually, to make clickhouse-local/clickhouse-client use
them too.

Changelog category (leave one):
- Not for changelog (changelog entry is not required)

v2: no need in `*_SYMBOLIZER_PATH` after #15069 (https://github.com/ClickHouse/ClickHouse/blob/123f389f1ae51f642b90950557321769da6caecc/docker/test/base/Dockerfile#L55)

### CODE REVIEW

**Review by alesapin — COMMENTED** [review]
Seems like now we have a lot of stderr in thread sanitizer build https://clickhouse-test-reports.s3.yandex.net/15063/5c58b9b1a75d59f242c50b5dd720b52c0bf29ad9/functional_stateless_tests_(thread)/test_run.txt.out.log. Probably need to add separate suppression for it.

File: docker/test/stateless/run.sh [hunk 1]
```diff
@@ -55,6 +55,12 @@ echo "ASAN_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environ
 echo "UBSAN_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environment
 echo "LLVM_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environment
 
+# /etc/environment will not be read in current shell
```
Comment (akuzm) [inline docker/test/stateless/run.sh:58]: Will it be read somewhere? Maybe just `export` right away, w/o `/etc/environment` at all?

Comment (azat) [reply docker/test/stateless/run.sh:58]: Yes, I guess initially it was done for clickhouse-server via service (there is `su` inside which will reset the env)

**Comment by azat** [comment]
>Seems like now we have a lot of stderr in thread sanitizer build https://clickhouse-test-reports.s3.yandex.net/15063/5c58b9b1a75d59f242c50b5dd720b52c0bf29ad9/functional_stateless_tests_(thread)/test_run.txt.out.log. Probably need to add separate suppression for it.

This is because of TSAN verbosity, this is superfluous for non-daemon binaries, removed

**Comment by alesapin** [comment]
@azat `01370_client_autocomplete_word_break_characters` failure seems to be related to changes?

**Comment by alesapin** [comment]
Real race catch in `01370_client_autocomplete_word_break_characters`

**Review by alesapin — APPROVED** [review]

**Comment by alesapin** [comment]
Waiting only for Functional stateless with thread sanitizer.

**Comment by alesapin** [comment]
Stress test failure is well known.

**Pull request merged**